### ISSUE TITLE
Consider adding GPLv3 licensing to Makefiles

### ISSUE THREAD

**Issue by jonalmeida** [minoca/os#3]
While browsing through the code, I noticed that the change in licensing was only for the source files and not the Makefiles. It would be nicer if they were included, if possible.

**Comment by evangreen** [comment]
Yes, we should do this. The fancy sed expression I created worked well for the .c, .h, .S, and .ck files, but didn't catch the Makefiles.

**Comment by jonalmeida** [comment]
I can try to put a patch together for this if you'd like.

**Comment by evangreen** [comment]
Yes, go ahead Jonathan.

**Comment by evangreen** [comment]
@jonalmeida Any progress here?

**Comment by jonalmeida** [comment]
@evangreen Sorry this took long. I ran into some issues but I finally managed to resolve them. I've created #52 with my patch. Here are the comments in my patch that someone might find useful in the future..

---

Since I'm using Mac OS X (10.11), I hard to learn the hard way that BSD and GNU sed treated newlines differently. I read about the difference between GNU and BSD sed thanks to a very helpful [SO comment][1] that explained the differences and solution.

The solution was to simply use GNU `sed` instead (pun intended) by installing it via Homebrew:
```bash
brew install gnu-sed
```

Using that along with `find`, allowed me to search through every file and call `gsed`:
```bash
find ./ -type f -exec sed -i -e 's/foo/bar/g' {} \;
```

Eventually I ended up with this regex:

```bash
find ./ -type f -exec gsed -i "s/Copyright (c) 2012 Minoca Corp. All Rights Reserved/Copyright (c) 2012 Minoca Corp.\n\n    This file is licensed under the terms of the GNU General Public License\n    version 3. Alternative licensing terms are available. Contact\n    info@minocacorp.com for details. See the LICENSE file at the root of this\n    project for complete licensing information.\n/" {} \;
```

Notes: I've replaced the copyright for each year with the GPL eqivalent with the same. i.e Copyright for 2012 gets a GPL since 2012, etc. 

[1]: http://stackoverflow.com/a/24276470/936067
